pkg/web/apps/apps_service.go

```diff
@@ -79,3 +81,17 @@ func (a *appsService) DisableAllAppVersionsByAppID(id string, message string) er
 
 	return a.repository.DisableAllAppVersionsByAppID(app.AppID, message)
 }
+
+func (a *appsService) BindingAppByApp(appId, name string) error {
+
+	app, err := a.repository.GetAppByAppID(appId)
+
+	if err != nil && err == models.ErrNotFound {
+		id := helpers.GetUUID()
+		return a.repository.CreateApp(id, appId, name)
+	} else if err != nil {
+		return err
+	} else {
```

Comment:
It would be better to split this logic into two seperate functions in my opinion to deal with the `create` and `undelete`.

Reply:
Done 👍 thank you for the review.